### DISCUSSION TITLE
Trying to update a TShock plugin and getting this error please help!!!

### DISCUSSION THREAD

**Discussion by sors89** [Pryaxis/TShock#1705] [Q&A] [answered]
![plsfix](https://user-images.githubusercontent.com/64196796/80084021-86143200-8580-11ea-8233-9d8debc5a985.png)

**Comment by QuiCM** [comment, marked as answer]
I would recommend checking how [other plugins](https://github.com/Pryaxis/TShock/blob/general-devel/TShockAPI/TSPlayer.cs#L1711) do it and experimenting a little to learn :)

**Reply by sors89** [reply]
Thanks <3
